examples/v2/chat2.nim

```diff
@@ -292,6 +294,12 @@ proc processInput(rfd: AsyncFD, rng: ref BrHmacDrbgContext) {.async.} =
     
     node.wakuStore.setPeer(parsePeerInfo(storenode))
 
+    if conf.swap:
+      var report =  node.wakuSwap.balance()
```

Comment:
What happens if you just print out node.wakuSwap.accounting? Should give you a list of peer id and current balance with each

Reply:
Yea sure. I considered this and I'm currently making an update